### ISSUE TITLE
Preorder Enriched Categories

### ISSUE THREAD

**Issue by jpfairbanks** [epatters/Catlab.jl#133]
Once we have a doctrine for preorders, we should be able to model V-enrichment. That is to say, take a presentation of X and assign to each Hom(x,y) an element of V. Then we can use matrix multiplication to interrogate X. This is based on Ch 2 of 7 sketches.

**Comment by epatters** [comment]
This would be cool. When I first read the title, I interpreted it as categories enriched **Preord**, the category of preorders. An example would be a (locally posetal) bicategory of relations. Then I realized you meant categories enriched in *a* preorder V.

**Comment by jpfairbanks** [comment]
Yeah I definitely mean enriched in a specific preorder like Bool (does a path exist?) or Cost to get Lawvere metric spaces.
